sys/tools/syscalls/scripts/init_sysent.lua

```diff
@@ -0,0 +1,221 @@
+#!/usr/libexec/flua
+--
+-- SPDX-License-Identifier: BSD-2-Clause
+--
+-- Copyright (c) 2024 Tyler Baxter <agge@FreeBSD.org>
+-- Copyright (c) 2023 Warner Losh <imp@bsdimp.com>
+-- Copyright (c) 2019 Kyle Evans <kevans@FreeBSD.org>
+--
+
+-- We generally assume that this script will be run by flua, however we've
+-- carefully crafted modules for it that mimic interfaces provided by modules
+-- available in ports.  Currently, this script is compatible with lua from
+-- ports along with the compatible luafilesystem and lua-posix modules.
+
+-- Setup to be a module, or ran as its own script.
+local init_sysent = {}
+local script = not pcall(debug.getlocal, 4, 1) -- TRUE if script.
+if script then
+    -- Add library root to the package path.
+    local path = arg[0]:gsub("/[^/]+.lua$", "")
+    package.path = package.path .. ";" .. path .. "/../?.lua"
+end
+
+local FreeBSDSyscall = require("core.freebsd-syscall")
+local util = require("tools.util")
+local generator = require("tools.generator")
+
+-- File has not been decided yet; config will decide file. Default defined as
+-- null
+init_sysent.file = "/dev/null"
+-- To align comments.
+local column = 80
+
+-- xxx need compat call count
+
+-- Should be the same as makesyscalls.lua generates, except that we don't bother
+-- to align the system call stuff... it's badly broken anyway and looks like crap
+-- so we're declaring that a bug and removing all that crazy book-keeping to.
+-- If we need to do it, and I hope we don't, I'll just create a string and do
+-- #str to figure out how many tabs to add
```

Comment:
I agree the current formatting is low value. If it's too ugly to make the output the same here, consider removing the alignment from makesyscalls.lua before the switch. The code change should be reviewable there whereas diffs to the generated files are impossible to review.

Reply:
I like this suggestion a lot. When this PR is closer to where it needs to be, I want to revisit this and open a PR to remove the aligning from makesyscalls.lua. I would like Kyle's input on this.